### PR TITLE
refactor: log add collection ready flag

### PR DESCRIPTION
**Pull request by wtt40122**
Add Toggle Switch for Immediate Log Collection and code clean
#371

### CODE REVIEW

File: ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/model/vo/KeywordPageParam.java
Comment (sadadw1) [inline ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/model/vo/KeywordPageParam.java:1]: New files must be marked with the Apache License.

Comment (wtt40122) [reply ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/model/vo/KeywordPageParam.java:1]: > New files must be marked with the Apache License.

ok

**Comment by goodjava** [comment]
What problem does this feature mainly solve? Can you describe it specifically?

**Comment by wtt40122** [comment]
> What problem does this feature mainly solve? Can you describe it specifically?

business team is looking for more granular control over the log collection process. They wish to have the autonomy to choose an ideal start time for log collection based on their readiness, ensuring that all aspects of the business systems, data processes, and monitoring mechanisms are fully prepared to ensure the efficiency and accuracy of log collection.

**Review by sadadw1 — APPROVED** [review]
+LGTM

**Review by wodiwudi — APPROVED** [review]
+LGTM

**Pull request merged**